trace_decoder/src/processed_block_trace.rs

```diff
@@ -211,8 +204,8 @@ fn check_receipt_bytes(bytes: Vec<u8>) -> anyhow::Result<Vec<u8>> {
 /// Note that "*_accesses" includes writes.
 #[derive(Debug, Default)]
 pub(crate) struct NodesUsedByTxn {
-    pub state_accesses: Vec<H256>,
-    pub state_writes: Vec<(H256, StateWrite)>,
+    pub state_accesses: Vec<Address>,
+    pub state_writes: Vec<(Address, StateWrite)>,
```

Comment:
Any reason why the switch hasn't been done as well to `accts_with_unaccessed_storage` and `self_destructed_accounts`. We could use the new `by_address` methods for them too I believe